packages/adminrouter/extra/src/test-harness/tests/conftest.py

```diff
@@ -33,7 +33,7 @@ def repo_is_ee():
 
 
 @pytest.fixture(scope='session')
-def mocker_s(repo_is_ee, syslog_mock, extra_lo_ips):
+def mocker_s(repo_is_ee, syslog_mock, extra_lo_ips, dns_server_mock_s):
```

Comment:
Comment added

> If we used a linter like pylint this would warn that dns_server_mock_s is an unused parameter.
We do not use pylint, this is a bigger effort to enforce it, and without it it will just bitrot (try grepping the source for pylint, I already played a bit with it here).